.github/workflows/ci.yml

```
@@ -19,7 +19,7 @@ name: wren / main
 
 on:
   push:
-    branches: [main, ci]
+    branches: [main, wren-package]
```

This was just temporary, right?

```suggestion
    branches: [main]
```

---

Yeah, I don't think it causes much harm, but I can also change it back. :)